src/containers/Tablet/Tablet.tsx

```diff
@@ -73,80 +130,108 @@ export const Tablet = () => {
-        const {TabletId, Overall, Leader} = tablet;
-
-        const externalLinks = [
-            {
-                name: `${DEVELOPER_UI_TITLE} - tablet`,
-                path: `/tablets?TabletID=${TabletId}`,
-            },
-        ];
-
+        const {TabletId, Overall} = tablet;
         return (
-            <div className={b()}>
+            <React.Fragment>
                 {error ? <ResponseError error={error} /> : null}
-                <div className={b('pane-wrapper')}>
-                    <div className={b('left-pane')}>
-                        <ul className={b('links')}>{externalLinks.map(renderExternalLinks)}</ul>
-                        <div className={b('row', {header: true})}>
-                            <span className={b('title')}>{i18n('tablet.header')}</span>
-                            <EntityStatus status={Overall} name={TabletId} />
-                            <a
-                                rel="noopener noreferrer"
-                                className={b('link', {external: true})}
-                                href={`${backend}/tablets?TabletID=${TabletId}`}
-                                target="_blank"
-                            >
-                                <Icon data={ArrowUpRightFromSquare} />
-                            </a>
-                            {Leader && <Tag text="Leader" type="blue" />}
-                            <span className={b('loader')}>{loading && <Loader size="s" />}</span>
-                        </div>
-                        <TabletInfo tablet={tablet} tenantPath={tenantName} />
-                        <TabletControls tablet={tablet} fetchData={refetch} />
-                    </div>
-                    <div className={b('rigth-pane')}>
-                        <TabletTable history={history} />
-                    </div>
-                </div>
-            </div>
+                <Flex gap={5} direction="column">
+                    <EntityPageTitle
+                        entityName={i18n('tablet.header')}
+                        status={Overall ?? EFlag.Grey}
+                        id={TabletId}
+                    >
+                        {loading && <Spin size="xs" className={b('loader')} />}
+                    </EntityPageTitle>
+                    <TabletControls tablet={tablet} fetchData={refetchTabletInfo} />
+                    <TabletInfo tablet={tablet} />
+                </Flex>
+            </React.Fragment>
         );
     };
     return (
-        <React.Fragment>
-            <Helmet>
-                <title>{`${id} — ${i18n('tablet.header')} — ${
-                    tenantName || queryClusterName || CLUSTER_DEFAULT_TITLE
-                }`}</title>
-            </Helmet>
-            {renderView()}
-        </React.Fragment>
+        <Flex gap={5} direction="column" className={b()}>
+            {renderHelmet()}
+            {renderPageMeta()}
+            <LoaderWrapper loading={loading && id !== tabletId} size="l">
```

Comment:
How can `id !== tabletId`, if you request tablet data by `tabletId`?

Reply:
You're right, this is nonsense...